index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+	
+<div class="divorved">
```

Comment:
Non-sensical class names will cause confusion when another teammate comes along and needs to update your code. Make sure you make it appropriate to either the content inside the element, or the purpose of the element (e.g. card-container)